tests/integration_tests/event/test_delete.py

```diff
@@ -17,6 +30,21 @@ def test_delete_event(client: TestClient, data_db: Session, user_header_token):
     )
 
 
+def test_delete_event_unfccc(
+    client: TestClient, data_db: Session, non_cclw_user_header_token
+):
+    setup_db(data_db)
+    response = client.delete(
```

Comment:
You could validate before you make the request that the event definitely exists?